test-packages/test-services/openapi/test-service/openapi/api/default-api.ts

```diff
@@ -232,11 +242,13 @@ export const DefaultApiFp = function(configuration?: Configuration) {
          * @summary Get entities
          * @param {string} [stringParameter] A parameter of type string
          * @param {number} [integerParameter] A parameter of type integer
+         * @param {string} [$dollarParameter] A parameter that contains $
+         * @param {string} [prefixSuffixParameter] A parameter that contains . in the middle
          * @param {*} [options] Override http request option.
          * @throws {RequiredError}
          */
-        async getAllEntities(stringParameter?: string, integerParameter?: number, options?: any): Promise<(axios?: AxiosInstance, basePath?: string) => AxiosPromise<Array<TestEntity>>> {
-            const localVarAxiosArgs = await DefaultApiAxiosParamCreator(configuration).getAllEntities(stringParameter, integerParameter, options);
+        async getAllEntities(stringParameter?: string, integerParameter?: number, $dollarParameter?: string, prefixSuffixParameter?: string, options?: any): Promise<(axios?: AxiosInstance, basePath?: string) => AxiosPromise<Array<TestEntity>>> {
```

Comment:
The `$` is still there in the default-api.

Reply:
is this the expected behavior that it is there?

Reply:
@marikaner , maybe you can comment it here.
I personally would not touch the `default-api.ts` since it's the vanilla openapi client?
Then we either keep the same naming strategy (use $dollarParameter) or align with our own convention.
I prefer the latter one, but it's open to be discussed.

Reply:
The $ is allowed in the beginning of a name of a parameter, therefore I don't see an issue. I would personally choose to keep the $, because it might be there for a reason and it does not hurt, but we would have to adjust the camelCase function for this (which is not that big a deal I guess). Not sure it is worth it.